apps/go-mod-validator/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "go-mod-validator",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Parses 'go.mod' file given as input, and verifies it's dependencies",
+  "engines": {
+    "pnpm": "9",
+    "node": "20"
+  },
+  "main": "out/index.js",
+  "scripts": {
+    "local": "GITHUB_STEP_SUMMARY=/dev/null ts-node src/index.ts",
+    "help": "ts-node --help",
+    "build": "esbuild src/index.ts --bundle --platform=node --target=node20 --tsconfig=./tsconfig.json --outfile=dist/index.js",
```

Comment:
Changed, oddly enough i cant get it to work properly when im running `nx` within the project dir, seems to be tied to the cwd. Got any ideas? Otherwise, running `nx go-mod-validator:build` works fine from the repo root